cassava_trait.obo

```diff
@@ -7594,6 +7594,139 @@ def: "Aroma like tha of sweet potato, Possibly like aroma of boiled, roasted or
+is_a: CO_334:0001000 ! Variables
+relationship: variable_of CO_334:0003051 ! fufu hardness in hand
+relationship: variable_of CO_334:0013009 ! texture by handfeel method
+relationship: variable_of CO_334:0103026 ! Hardness scale 0-10
+created_by: Ugo Chijioke
+
+[Term]
+id: CO_334:0003045
+name: Fufu stretchability scale 0-10
+namespace: CassavaTrait
+def: "Measures the ability of fufu to stretch when pulled apart with the fingers" []
+is_a: CO_334:0001000 ! Variables
+relationship: variable_of CO_334:0003052 ! fufu stretchability in hand
+relationship: variable_of CO_334:0013010 ! texture by handfeel method
+relationship: variable_of CO_334:0103027 ! Stretchability scale 0-10
+created_by: Ugo Chijioke
+
+[Term]
+id: CO_334:0003046
+name: Fufu odour scale 0-10
+namespace: CassavaTrait
+def: "Measures the odour fufu  smelt with nose when it is just cooked" []
+is_a: CO_334:0001000 ! Variables
+relationship: variable_of CO_334:0003053 ! fufu odour
+relationship: variable_of CO_334:0013011 ! odour method
+relationship: variable_of CO_334:0103028 ! Fufu odour scale 0-10
+created_by: Ugo Chijioke
+
+[Term]
+id: CO_334:0003047
+name: fufu surface colour
+namespace: CassavaTrait
+def: "Colour of cooked fufu being as white as a flip chart paper or as as off white as soy milk colour, or as grey as skin of sardine fish, or as light as cement colour, or as cream coloured as peak evaporated liquid milk, or as light yellow as cooked egg york, or as yellow as skin of ripe orange, or as brown in colour as whole wheat meal" []
+synonym: "FufSurfCol" EXACT []
+is_a: CO_334:0103017 ! Appearance trait
+
+[Term]
+id: CO_334:0003048
+name: fufu smoothness in hand
+namespace: CassavaTrait
+def: "Absence of lumps" []
+synonym: "FufHdSmoth" EXACT []
+is_a: CO_334:0103019 ! Texture in hand trait
+
+[Term]
+id: CO_334:0003049
+name: fufu stickiness in hand
+namespace: CassavaTrait
+def: "This is when the fufu sticks or adhere to the fingers when it is touched with fingers or during moulding into bolus for swallowing" []
+synonym: "FufHdStick" EXACT []
+is_a: CO_334:0103019 ! Texture in hand trait
+
+[Term]
+id: CO_334:0003050
+name: fufu moldability in hand
+namespace: CassavaTrait
+def: "Ability of the fufu to stick together easily without disintegrating or cohesive and being easy to mould." []
+synonym: "FufHdMol" EXACT []
+is_a: CO_334:0103019 ! Texture in hand trait
+
+[Term]
+id: CO_334:0003051
+name: fufu hardness in hand
+namespace: CassavaTrait
+def: "Softness is when it is quite easy to compress the fufu sample, while hardness is when force is used to compress the sample or takes effort to compress the sample" []
+synonym: "FufHdHard" EXACT []
+is_a: CO_334:0103019 ! Texture in hand trait
+
+[Term]
+id: CO_334:0003052
+name: fufu stretchability in hand
+namespace: CassavaTrait
+def: "Ability of fufu to stretch when pulled apart with fingers" []
+synonym: "FufHdStret" EXACT []
+is_a: CO_334:0103019 ! Texture in hand trait
+
+[Term]
+id: CO_334:0003053
```

Comment:
should this be consolidated with 'fufu smell'? 
https://github.com/Planteome/CO_334-cassava-traits/blob/3b05da7d5a44d0b02f8f74702c66329f5f4c78c4/cassava_trait.obo#L6496

Reply:
I'll wait for the commit and then will merge this and update Cassavabase. Thanks!